llvm/lib/CodeGen/MachineCopyPropagation.cpp

```diff
@@ -1053,7 +1053,7 @@ void MachineCopyPropagation::BackwardCopyPropagateBlock(
     // Ignore non-trivial COPYs.
     std::optional<DestSourcePair> CopyOperands =
         isCopyInstr(MI, *TII, UseCopyInstr);
-    if (CopyOperands && MI.getNumOperands() == 2) {
+    if (CopyOperands) {
```

Comment:
Obviously the immediate is irrelevant... but can we have implicit register operands here?  Do we care if we do?  It looks like isCopyInstrImpl() doesn't check for them.

Reply:
I have a patch to fix that stuck on its 6th revert